local-scripts/print-json.py

```diff
@@ -14,6 +14,14 @@
 def set_file_indent(file):
     # This code breaks in 2030 and.or after we assign 1 million GSDs per year
     file_name = os.path.basename(file)
+
+    # We match 2021 through 2029, 1 million and up so that's guaranteed GSD space only
+    # GSD bot uses 2 space for indent
+    # The CVE Bot uses the default 4 spaces, hence the need for checking
+    # We can ismply read the file, scheck the second line and count spaces because
```

Comment:
We can't simply read the file, check the second line and count space because

Reply:
because jq.

Reply:
We **can ismply** read the file, **scheck** the second line and count spaces because
We **can't simply** read the file, **check** the second line and count space because

Bold for clarity on what needs changed.